features/users.feature

```diff
@@ -467,13 +467,13 @@ Feature: Users
                 "session": "auth-challenge-session"
             }
             """
-        Then I should receive the following JSON response with status "501":
+        Then I should receive the following JSON response with status "400":
```

Comment:
The scenario description here no longer matches whats being tested.

Reply:
I had to leave one mistake to ensure that you were checking